tests/URLConstructorTest.php

```diff
@@ -37,25 +39,8 @@ public function getTestData()
      */
     public function testUrl($expected)
     {
-        // Skip over comments in the json file.
-        if (is_string($expected)) {
-            return;
-        }
-
-        $shouldFail = property_exists($expected, 'failure') &&
```

Comment:
This is probably where coveralls has an issue since the code path in the URL object's constructor that throws an exception is no longer being tested. Adding a new test that expects an exception of `\Rowbot\URL\TypeError` to be thrown for all data that has the `failure` property should satisfy coveralls.